docs/assignment1.md

```diff
@@ -1,62 +1,52 @@
+Authors: Serghei Mihailov, Bjorn Keyser, Yael Goede, Milos Delgorge
 
-### Functional features
+The name of the project is Snippo. The system we will be implementing will be an implementation based on the code snippets manager. The goal of the code snippets manager is to create a system to manage snippets of source code. The system will simplify the managing of snippets by providing formats and functionalities that will let the user search, edit, and create source code snippets. The system will be able to adapt to new languages introduced by the user, hereby increasing the potential user group to all programming language users. The snippets manager will be hosted locally on the machine. The user will be able to interact with the manager through the shell. Here the user can use a crud api to manage the available snippets. The system will also provide a search command that will make retrieving snippets easier. The search command will use other features, for example contents, tags and metadata to optimise the search speed. 
 
-As a preamble to the table, you can discuss the main line of reasoning you applied for defining the functional features provided in the table.
+Existing snippet manager applications for inspiration:
+1. Gisto: https://www.gistoapp.com/features
+2. Github Gists: https://developer.github.com/v3/gists/.
 
-| ID  | Short name  | Description  |
-|---|---|---|
-| F1  | Tags | Code snippets can be tagged via freely-defined labels called tags  |
-| F2  | Commands  | The player can control the main character by issuing command-line commands following this syntax: `command-name [target-objects]*`. The available `command-names` are the following: <br/> - move: ... <br/> - use: ... <br/> - inspect: ... <br/> |
-| F3  | Movements  | The main character shall move freely in the environment according  |
-| F4  | ... | ... |
+## Features:
 
-### Quality requirements
-Author(s): `name of the team member(s) responsible for this section`
+Author(s):Serghei Mihailov, Bjorn Keyser, Yael Goede, Milos Delgorge
 
-As a preamble to the table, you can discuss the main line of reasoning you followed for coming up with the quality requirements listed below.
+| **ID** | **Short name** | **Description** |
+| --- | --- | --- |
+| F1 | API to Create/Read/Update/Delete snippets | Users should be able to easily access edit, create and delete snippets by calling the tool from the CLI using arguments. |
+| F2 | Snippet editor | Users should be able to edit and save snippets via a basic Vim-like CLI editor with visual/insert/replace modes. |
+| F3 | Tags | Users should be able to add tags to snippets by which will make finding snippets easier. This should include language and whether the snippet is "starred". |
+| F5 | Search/Filter | Users can search for their snippets by content, tags, description, title, language. Like snippo -s "language:java tags:tag1,tag2 searchterm". |
```

Comment:
You could merge these into just "Filter" since that is the functionality that's being described here.